R/tm_variable_browser.R

```diff
@@ -871,24 +872,33 @@ var_summary_table <- function(x, numeric_as_factor, dt_rows, outlier_definition)
 #' @param var vector of any type to be plotted. For numeric variables it produces histogram with
 #' density line, for factors it creates frequency plot
 #' @param var_lab text describing selected variable to be displayed on the plot
-#' @param numeric_as_factor \code{logical} should the numeric variable be treated as a factor
-#' @param display_density \code{logical} Should density estimation be displayed for numeric values?
-#' @param remove_NA_hist \code{logical} Should \code{NA} values be removed for histogram of factor like variables.
+#' @param wrap_character (`numeric`) Number of characters at which to wrap text values of `var`.
+#' @param numeric_as_factor (`logical`) should the numeric variable be treated as a factor
+#' @param display_density (`logical`) Should density estimation be displayed for numeric values?
+#' @param remove_NA_hist (`logical`) Should (`NA`) values be removed for histogram of factor like variables.
 #' @param outlier_definition If 0 no outliers are removed, otherwise
 #'   outliers (those more than outlier_definition*IQR below/above Q1/Q3 be removed)
-#' @param records_for_factor \code{numeric} if the number of factor levels is >= than this value then
+#' @param records_for_factor (`numeric`) if the number of factor levels is >= than this value then
 #'   a graph of the factors isn't shown, only a list of values.
+#'
 #' @return plot
 #' @keywords internal
 plot_var_summary <- function(var,
                              var_lab,
+                             wrap_character = NULL,
                              numeric_as_factor,
                              display_density = is.numeric(var),
                              remove_NA_hist = FALSE, # nolint
                              outlier_definition,
                              records_for_factor,
                              ggplot2_args) {
+  checkmate::assert_character(var_lab)
+  checkmate::assert_numeric(wrap_character, null.ok = TRUE)
+  checkmate::assert_logical(numeric_as_factor)
   checkmate::assert_flag(display_density)
+  checkmate::assert_logical(remove_NA_hist, null.ok = TRUE)
```

Comment:
Why allow `NULL`?

Reply:
The `remove_NA_hist` value is present solely when NA values exist. Otherwise, the userInput for `remove_NA_hist` is undefined, often carrying a NULL value in numerous instances.

Reply:
Ah, that's why the `isTRUE` later on, got it.